src/commands/showC.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { createCommand } from '../utils/helpers';
+
+const description = `Show copyright notice.
+
+	Examples:
+	- show c
+`;
+
+const copyright = `
+Lisky  Copyright (C) 2017  Lisk Foundation
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+`;
+
+export const actionCreator = () => () => Promise.resolve({ copyright });
+
+const showCopyright = createCommand({
+	command: 'show c',
```

Comment:
`show copyright` with alias `show c`. Can we also name the file `showCopyright`?